ui/ui-frontend/projects/collect/src/app/collect/archive-search-collect/archive-search-criteria/components/criteria-search/criteria-search.component.html

```diff
@@ -238,7 +238,7 @@
                 }}:{{
                   criteriaValue.valueTranslated
                     ? ('COLLECT.SEARCH_CRITERIA_FILTER.' + getCategoryName(criteriaVal.category) + '.' + criteriaValue.label | translate)
-                    : criteriaValue.label
+                    : (getCriteriaLabel(criteriaVal.key, criteriaValue) | truncate: 50)
```

Comment:
Ne pas tronquer dans le tooltip

```suggestion
                    : getCriteriaLabel(criteriaVal.key, criteriaValue)
```